discv5-utp.md

```diff
@@ -27,13 +27,24 @@ Bob, upon sending the `Data` message containing the `connection_id` will
 When this new connection is opened, Alice can then read the bytes from the stream
 until the connection closes.
 
+The `connection_id` send over the sub-protocol response message is the
+`connection_id_send` value for node sending this response, and thus the `connection_id_recv`
+value for initiator of the uTP connection.
+
 ## BEP29
 
 https://www.bittorrent.org/beps/bep_0029.html
 
 The uTP protocol as specified in BEP29 defines the packet structure and logic for handling packets.
 
-The only change we make to BEP29 is that rather than using the full UDP packet for uTP packets, we instead use the TALKREQ `payload` for the uTP packet.
+The main difference with BEP29 is that, instead of a raw UDP packet, the Discovery v5 `TALKREQ` message is used as transport.
+
+Additionally, following deviations from the uTP specification or reference implementation are applied:
```

Comment:
When you asked about it on discord I forgot to mention one another deviation from spec which reference implementation also do.

When receiving  `SYN-ACK` client should initialise its `ackNr` to `synAck.seqNr - 1` instead of `synAck.seqNr`  (in code: https://github.com/status-im/nim-eth/blob/master/eth/utp/utp_socket.nim#L826)

Reply:
Thanks, I've added that to the list.